deploy/deployment.yml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ktr
+  name: token-refresher
+  namespace: catalog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: token-refresher
+  template:
+    metadata:
+      labels:
+        app: token-refresher
+    spec:
+      containers:
+      - name: refresher
+        env:
+        - name: KTR_SECRETNAME
+          value: "bearer-creds"
+        - name: KTR_SECRETNAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: KTR_SECRETKEY
+          value: "token"
+        - name: KTR_OIDC_TOKENURL
+          value: "https://auth.vshn.net/token"
+        - name: KTR_OIDC_CLIENTID
+          valueFrom:
+            secretKeyRef:
+              name: ktr
+              key: id
+        - name: KTR_OIDC_CLIENTSECRET
+          valueFrom:
+            secretKeyRef:
+              name: ktr
+              key: secret
+        image: docker.io/vshn/kube-token-refresher:v1.0.0
+        imagePullPolicy: Always
+        resources:
+          limits:
+            cpu: 100m
+            memory: 30Mi
+          requests:
+            cpu: 100m
+            memory: 10Mi
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      serviceAccountName: ktr
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ktr
+  namespace: catalog
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: service-ktr
+  namespace: catalog
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ktr
+subjects:
+- kind: ServiceAccount
+  name: ktr
+  namespace: catalog
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ktr
```

Comment:
Please don't use abbreviations in such resources. It makes operations unnecessary hard or confusing for those unfamiliar with the abbrev.